### PR TITLE
Fix #83: Add Council events page

### LINKED ISSUE #83: Add events page for assoc. events
As discussed during the Council meeting today, there should be an events page where the member associations can add their events.

### PR DESCRIPTION
**Pull request by sdruskat**
Fixes issue #83:

- Add a simple page for events run by member associations
- Link to the events page from the Council landing page

### CODE REVIEW

File: council.md [hunk 1]
```diff
@@ -52,3 +52,7 @@ The Council has a [*Code of Conduct*](./council/code-of-conduct.md), and you are
 | Nordic-RSE  | Radovan Bast | radovan.bast@uit.no |
 | US-RSE  | Daniel S. Katz | d.katz@ieee.org |
 | US-RSE  | Nicole Brewer | brewer36@purdue.edu |
+
+## Events
+
+Have a look at the [Council Events](council/events.html) page for a list of events run by member associations.
```
Comment (danielskatz) [inline council.md:58]:
```suggestion
Have a look at the [Council Events](council/events.html) page for a list of events run by member associations of potential international interest. Note that, in addition, most associations maintain their own complete list of their events.
```

File: council/events.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: The International Council of RSE Associations - Events
+hidetitle: false
+---
+
+The following is a list of events run by member associations.
```
Comment (danielskatz) [inline council/events.md:7]:
```suggestion
The following is a list of events run by member associations of potential international interest. Note that, in addition, most associations maintain their own complete list of their events.
```

**Comment by sdruskat** [comment]
> Should we also add a place where member associations can provide a link to their events/calendars? For example, for US-RSE, https://us-rse.org/events/

Yes, I woke up today thinking that. Will create a place for this on the events page.

**Review by danielskatz — APPROVED** [review]
lgtm

**Comment by danielskatz** [comment]
Should we also add Events to the "menu bar"?

**Comment by danielskatz** [comment]
And maybe add something about events on https://researchsoftware.org?

**Comment by sdruskat** [comment]
> @sdruskat - I'm going to open a new issue about this

Sorry, @danielskatz, had no time to fix this yet. A new issue is better for visibility, thanks.

**Pull request merged**